test/integration/crd_rw_test.go

```diff
@@ -68,11 +67,11 @@ var _ = Describe(fmt.Sprintf("%s,%s: Basic CRD Read-Write", focusAws, focusAzure
 	)
 
 	AfterEach(func() {
-		result := CurrentGinkgoTestDescription()
-		if result.Failed {
+		result := CurrentSpecReport()
+		if result.Failed() {
```

Comment:
not sure if this return failure of the current test or any test failures from specs?

Reply:
Solved.